app/javascript/selectors/detailSelector.js

```diff
@@ -67,7 +67,7 @@ export const disableActionButton = state => {
   return safeGet(state, 'fetchDetails.disableActionBtn')
 }
 
-export const checkEditDisable = state => !safeGet(state, 'fetchDetails.details.records.edit_details.editable')
+export const isUserEditable = state => safeGet(state, 'fetchDetails.details.records.edit_details.editable') || false
```

Comment:
```suggestion
export const isUserEditable = state => safeGet(state, 'fetchDetails.details.records.edit_details.editable', false)
```
I think the semantics are the same, where safeGet gives you false if the expression can't be found.

Reply:
Yeah instead i'm grabbing the value and making use of it! Both works as well